### PR TITLE
Updates for buttervolume 3.0

### PR DESCRIPTION
**Pull request by ccomb**

### CODE REVIEW

File: consul/handler.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 DTFORMAT = "%Y-%m-%dT%H%M%S.%f"
 DEPLOY = '/deploy'
 CADDYLOGS = '/var/log'
+BTRFSDRIVER = os.environ['BTRFSDRIVER']
```
Comment (petrus-v) [inline consul/handler.py]: could be nice to provide a default value likes:

``
BTRFSDRIVER = os.environ.get('BTRFSDRIVER', 'anybox/buttervolume:latest')
``

**Comment by ccomb** [comment]
I've successfully made a deploy and a switch to another node with this version. Buttervolume now works from inside consul, and the send/receive also works.  It needs buttervolume 3.4.

File: consul/handler.py (hunk 1 above)
Comment (petrus-v) [reply consul/handler.py]: You take care of this comment  above but you should remove this line which make [travis unhappy](https://travis-ci.org/mlfmonde/cluster/builds/377346022#L1139)

Actually I apologies I think I've made this duplication trough [that PR](https://github.com/ccomb/cluster/pull/1)

**Comment by petrus-v** [comment]
Thanks about that one. LGTM. Test passed against https://github.com/mlfmonde/cluster_lab/pull/6

**Pull request merged**